src/cli/subcommands/chain_cmd.rs

```diff
@@ -151,15 +160,43 @@ fn maybe_confirm(no_confirm: bool, prompt: impl Into<String>) -> anyhow::Result<
     }
 }
 
-/// Print the first `n` tipsets from the head (inclusive).
-async fn print_chain_head(client: &rpc::Client, n: u64) -> anyhow::Result<()> {
+#[derive(Debug, serde::Serialize)]
+struct TipsetInfo {
+    epoch: u64,
+    cids: Vec<String>,
+}
+
+/// Collects `n` tipsets from the head (inclusive) and returns them as a list of
+/// `TipsetInfo` objects.
```

Comment:
Let's have proper Rust doc [linking](king-to-items-by-name.html#linking-to-items-by-name).
```suggestion
/// Collects `n` tipsets from the head (inclusive) and returns them as a list of
/// [`TipsetInfo`] objects.
```

Reply:
Done.